### ISSUE TITLE
Optimization in serial?

### ISSUE THREAD

**Issue by esmuigors** [SimonBlanke/Hyperactive#51]
Dear SimonBlanke,

First of all, thank You for this wonderful project!

My question is: can I somehow specify the optimizers to NOT run in parallel? My function is actually a call to external (parallelized) program, which furthermore cannot be run in multiple instances on the same machine (it is a commercial program). So I would like to prevent the optimizer from initiating next calculations while the first one is not finished.

Best regards,
Igors

**Comment by esmuigors** [comment]
Dear @SimonBlanke 

I am very sorry for hastily opening this issue (though it was the opportunity to say those kind words :) ). It turned out to be a problem completely unrelated to either gradient_free_optimizers or the Hyperactive... I didn't look at the modification times of the files, then it became obvious :'-) What a shame on me...

I hope that You answer will be useful for someone else at least :)

Best wishes,
Igors

**Comment by SimonBlanke** [comment]
Hello @esmuigors,

no Problem! I think the question and answer is still useful for others.
If you have more questions don't hesitate to open another issue.
